src/main/resources/db/migration/V1_5__create_courthouse_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE moj_courthouse (
+  moj_crt_id                INTEGER                         GENERATED ALWAYS AS IDENTITY
+, courthouse_name           CHARACTER VARYING               NOT NULL
+, code                      SMALLINT
+, created_date_time         TIMESTAMP with TIME ZONE     NOT NULL
+, last_modified_date_time   TIMESTAMP with TIME ZONE     NOT NULL
+, CONSTRAINT moj_courthouse_pkey PRIMARY KEY (moj_crt_id)
+);
+
```

Comment:
Please can you add the sequence for the Id column
`CREATE SEQUENCE IF NOT EXISTS moj_crt_seq`